src/components/Calculator.js

```diff
@@ -5,142 +5,135 @@ import 'bootstrap/dist/js/bootstrap.min';
+              9
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 mathematical-operator border  border-1" onClick={() => { results(element, 'x', setElement); }}>
+              x
+            </button>
+          </div>
+          <div className="row">
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '4', setElement); }}>
+              4
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '5', setElement); }}>
+              5
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '6', setElement); }}>
+              6
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 mathematical-operator border  border-1" onClick={() => { results(element, '-', setElement); }}>
+              -
+            </button>
+          </div>
+          <div className="row">
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '1', setElement); }}>
+              1
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '2', setElement); }}>
+              2
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '3', setElement); }}>
+              3
+            </button>
+
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1 mathematical-operator" onClick={() => { results(element, '+', setElement); }}>
+              +
+            </button>
+          </div>
+          <div className="row">
+            <button type="button" className="col-6 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '0', setElement); }}>
+              0
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1" onClick={() => { results(element, '.', setElement); }}>
+              .
+            </button>
+            <button type="button" className="col-3 d-flex justify-content-center align-items-center py-3 border  border-1 mathematical-operator" onClick={() => { results(element, '=', setElement); }}>
+              =
+            </button>
```

Comment:
- [ ] **[Optional] Please consider refactoring this code so that your code follows the DRY approach.**

Each line of code is creating the same component. You can create a collection of data to be passed to each button component and then you can iterate through that collection.